script/watch.js

```diff
@@ -0,0 +1,103 @@
+const chokidar = require('chokidar');
+const CLIEngine = require('eslint').CLIEngine;
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+const esLinter = new CLIEngine();
+const sasslinter = require('sass-lint');
+
+const chalk = require('chalk');
+const path = require('path');
+const srcWatcher = chokidar.watch(['./src/**/*.jsx', './src/**/*.js', './src/**/*.scss']);
+
+let running = false;
+let restart = false;
+
+function formatter(eslintResults) {
```

Comment:
Only including errors and their file name, line number, and error message. Can include whatever and format it however. 
<img width="643" alt="screen shot 2018-03-23 at 00 08 22" src="https://user-images.githubusercontent.com/4998130/37814174-53d41ab8-2e2e-11e8-8ec6-9d1a726f4a8d.png">